COORDINATION.md

```diff
@@ -6,9 +6,81 @@ Reference [BOOTSEQ.md](./BOOTSEQ.md)
 
 - what problems we have today with PCIe, timeouts, errors, retries,...
 
-## Out-band via BMC
+## Out-band via platform BMC
 
-- i2c
-- ncsi
-- usb
-- others...
+### PLDM State sensors
+Useful State definitions
```

Comment:
let's add physical layer here... where PLDM can be protocol on top... i2c, usb, others?

Reply:
in next PR